src/main/java/mate/academy/dto/user/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.dto.user;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Data;
+import lombok.experimental.Accessors;
+import mate.academy.lib.FieldMatch;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+@FieldMatch(first = "password", second = "repeatPassword", message = "Passwords must match")
+@Accessors(chain = true)
+public class UserRegistrationRequestDto {
+    @NotNull
```

Comment:
for String use `@NotBlank` (check all places)
```suggestion
    @NotBlank
```